hail_search/queries/gcnv.py

```diff
@@ -84,11 +85,44 @@ def get_allowed_sv_type_ids(self, sv_types):
             type.replace(self.SV_TYPE_PREFIX, '') for type in sv_types if type.startswith(self.SV_TYPE_PREFIX)
         ])
 
+    def _annotate_families_table_annotations(self, families_ht, annotations_ht, is_comp_het=False):
```

Comment:
For gCNV data, we have an entry-level gene_ids annotation, as the breakpoints for the SV in the annotation table can be different than for an individual call. The changes in this file ensure that gene and annotation filters only consider the genes present in the entries, and not all the gene. This function is added because we *also* recently added the logic to do an annotation-first search if genes are preent, which means at the time we do the gene and annotation filtering we don;t have the entries available, so we need to add a post-processing step to re-filter based on the filtered genes in that case